app/models/event.rb

```diff
@@ -16,9 +19,13 @@ class Event < ActiveRecord::Base
   validates :slug, uniqueness: true
   validate :invitability, if: :invitable?
   validates :coach_spaces, :student_spaces, numericality: true
+  validate :sponsors_uniqueness
+  validate :bronze_sponsors_uniqueness
+  validate :silver_sponsors_uniqueness
+  validate :gold_sponsors_uniqueness
 
   before_save do
-    begins_at = Time.parse(self.begins_at)
+    begins_at = Time.zone.parse(self.begins_at)
```

Comment:
I think this should be tested and critiqued in a different PR, just in case it's that say for a reason

Reply:
sorry I have no idea how this ended up in there, I imagine I corrected it with `rubocop` by mistake, let me restore it as it was.